### ISSUE TITLE
Error while submitting Loved maps

### ISSUE THREAD

**Issue by xasuma** [ppy/osu-stable-issues#486]
There are a some loved maps (https://osu.ppy.sh/beatmapsets/90893#osu/246322 this as an example) that do not allow users to submit scores on them. The moment you finish the map you will get an error message.

All the scores in them were from users who submitted scores shortly after it becoming loved, for some reason they worked fine for a while after becoming loved but after a certain amount of time they stop taking score submissions.

**Comment by peppy** [comment]
Could you please include your network.log (if using cutting edge) and what error message you see? Thanks!

**Comment by Coppertine** [comment]
iirc, that's the 2b auto fail. If it says that it can't process the score.
Basically what's happening right now is that 2b has a failsafe so that when a person autofails, they instead get the results screen with an error stating that it can't process the score.

**Comment by xasuma** [comment]
Coppertine is correct. Weirdly enough when I swapped over to cutting edge, the map I mentioned above did submit, and then I switched back to stable and got a slight better score and it also submitted again. 

After testing many maps I came to the conclusion that there are some maps that only allow submissions with the EZ mod such as: 
(laundry diff) https://osu.ppy.sh/beatmapsets/639281#osu/1417193
(3ww diff) https://osu.ppy.sh/beatmapsets/613989#osu/1295309
(nino's aspire diff) https://osu.ppy.sh/beatmapsets/537048#osu/1529760

But there are also maps that do not take submissions no matter what, such as: 
(Koa%wer diff) https://osu.ppy.sh/beatmapsets/639281#osu/1355907
(x.x~ diff) https://osu.ppy.sh/beatmapsets/598905#osu/1265925

I took a look at the network.log and no errors in there, all maps I tried were successful in the log. (My network.log dissapeared when I changed back to stable, oops)

It seems that all the problematic maps have at least one section of 2 notes playing at one time, and that is causing the issues as far as I can tell.

**Comment by Coppertine** [comment]
this can be simular to #136 but affecting aspire like mapsets. All diffs given as examples have some form of 2B and can autofail. The issue isn't 2 notes at once, it's an issue of sliders and circles not behaving correctly. It's more like the focus of the slider isn't set once enough objects over a slider is hit. 
It's best to leave this as is since 2B and the autofail system has been one of the longest things that can't really be fixed in the first place

**Comment by peppy** [comment]
Indeed. Not sure if there's a simple fix for this; may be something we just have to accept is broken until a better solution comes with lazer.
